apps/ledger-live-mobile/src/components/FloatingDebugButton.tsx

```diff
@@ -88,6 +88,7 @@ const FloatingDebugButton: React.FC<Props> = ({
           {React.isValidElement(Icon) ? (
             Icon
           ) : (
+            /* @ts-expect-error TS 5 can't seem to be able to prove this is a react comopnent here */
```

Comment:
I don't know how to prove to TypeScript that is still valid. if no one knows we will go with the present tradeoff :'(